tasks/README.md

```diff
@@ -25,7 +25,20 @@ Firestore Tasks collection data model
   isCollapsed: boolean,
   createdBy: string,
   assignee: string,
-  participants: [userId, userId]
+  participants: [userId, userId],
+  github: {
+    issue: {
+      status: string,
+      assignee: string,
```

Comment:
How is this assignee different from the root level assignee?

And is status like something [open, closed, etc]?

Reply:
- This assignee refers to GitHub issue assignee.
- According to the docs, the status is always `closed` or `open`

Reply:
Should we comment on that? the github assignee part
What do you think?

Reply:
Done, I've included the two values beside the `status` itself.